src/Text.Analyzers/UnitTests/Text.Analyzers.UnitTests.csproj

```diff
@@ -2,6 +2,9 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>net46</TargetFramework>
+
+    <!-- Do not directly await a task-->
+    <NoWarn>$(NoWarn);CA2007</NoWarn>
```

Comment:
Move this entry to https://github.com/dotnet/roslyn-analyzers/blob/master/eng/Analyzers_NonShippingRules.ruleset? Actually, my other PR does it here: https://github.com/dotnet/roslyn-analyzers/pull/2116/files#diff-b7297f6c12b0c22035d69cf58a19ae2fR15

Reply:
I also did it in #2119